src/calibration/calibration.cpp

```diff
@@ -0,0 +1,57 @@
+#include "hyrise.hpp"
+#include "logical_query_plan/lqp_translator.hpp"
+#include "logical_query_plan/mock_node.hpp"
+#include "scheduler/operator_task.hpp"
+#include "types.hpp"
+
+#include "calibration_benchmark_runner.hpp"
+#include "calibration_lqp_generator.hpp"
+#include "calibration_table_generator.hpp"
+#include "operator_feature_export.hpp"
+#include "table_feature_export.hpp"
+
+using namespace opossum;  // NOLINT
+
+int main() {
+  auto table_config = std::make_shared<TableGeneratorConfig>(TableGeneratorConfig{
+      {DataType::Double, DataType::Float, DataType::Int, DataType::Long, DataType::String, DataType::Null},
+      {EncodingType::Dictionary},
+      {ColumnDataDistribution::make_uniform_config(0.0, 1000.0)},
+      {100000},
+      {1500, 3000, 6000, 10000, 20000, 30000, 60175, 25, 15000, 2000, 8000, 5, 100}});
```

Comment:
I would put these (and the ones above) into variables, e.g., `constexpr auto row_counts ...` to make it more descriptive